.github/workflows/linters.yml

```diff
@@ -0,0 +1,36 @@
+name: Linters
+
+on: pull_request
```

Comment:
- [ ] Your linters are not passing all the test. Kindly fix your linter errors 😊

Reply:
@HAMMAS-SALEEM Bro. Hope you are fine. if you have checked the error you will find that it's a ruby version error

Reply:
Would you suggest me how to fix it, please? because I'm not getting the errors on my PC. Its only on github

Reply:
Hey @Bateyjosue sorry for the late response.I can see you have already fixed the errors :blush: That's really a great thing which you did. Keep doing great things like that. Happy Coding :+1: